README.md

```diff
@@ -1,8 +1,6 @@
 # Has Many Relationships
 
-When connecting to your pg database, connect from this project directory.
-
-To run sql statements from an external file, use the \i [filepath] command.
+To run sql statements from an external file, load them into DBeaver and run them from there.
```

Comment:
Should we use references to an agnostic tool so that we don't have to continue to change copy w/ tools. Use your RDBMS client.... or something like that.